website/views/shared/profiles/stats.jade

```diff
@@ -28,7 +28,8 @@ unless mobile
     tr(ng-repeat='(k,v) in profile.items.gear.equipped', ng-init='piece=Content.gear.flat[v]', ng-show='piece')
       td
         strong {{piece.text()}}:&nbsp;
-        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{stat.toUpperCase()}}&nbsp;
+        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{env.t(stat)}}&nbsp;
+        span(ng-hide='piece.str || piece.con || piece.per || piece.int')=env.t('noBenefit')
```

Comment:
I'm not sure I like this bit. Plus, if you aren't wearing a piece of equipment, it looks like this (note the "No Helm" and "No Shield Hand Equipment":

![screen shot 2015-06-27 at 9 25 52 pm](https://cloud.githubusercontent.com/assets/2916945/8394904/2e8d249a-1d13-11e5-999f-81ffb0efd7de.png)

Which seems odd to me.